### PR TITLE
First pass at R function to write elastic net results

### PR DESCRIPTION
**Pull request by khoulahan**
Updated schema so compatible with RProtoBuf and added messages to document elastic net coefficients. Also added optional option to specify alpha used. 

Generated R function to write elastic net results from glmnet R object. 

Created examples folder with training data and elastic net results.

### CODE REVIEW

File: R/populate_ML_protobuf.R [hunk 1]
```diff
@@ -0,0 +1,122 @@
+### populate_ML_protobuf.R ########################################################################
+# document elastic net run as protocol buffer according to provided schema 
+###################################################################################################
+library(glmnet);
+# read in sample feature matrix 
+sample.feature.matrix <- read.delim(
+	'/Users/user/Documents/Pathway/HallmarkGeneSet/2015-07-20_AllSummaryMetrics.txt',
```
Comment (kellrott) [inline R/populate_ML_protobuf.R]: Hard coded paths!?!?!?!

**Comment by khoulahan** [comment]
Whoops, committed an old version. Committed most recent version now - no hard-coded paths!

File: proto/ml_schema.proto [hunk 2]
```diff
@@ -36,31 +37,29 @@ message ModelStructure {
...
-    int32 AboveChild = 1;
-    int32 BelowChild = 2;
-    string SplitVariable = 3;
-    double SplitValue = 4;
-    bool LeafNode = 5;
-    bool Label = 6;
-  }
-  repeated DecisionNode Nodes = 1;
-}
-
```
Comment (kellrott) [inline proto/ml_schema.proto:57]: Why did you remove the DecisionTree code?

**Comment by khoulahan** [comment]
I didn't mean too. Must have forgot to update when I committed again, my bad ...
